### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

**Comment by vmunishwar** [comment]
Good catch on the typos - LGTM

**Comment by hezhizhen** [comment]
@vmunishwar @cppforlife

**Review by jtigger — APPROVED** [review]
@hezhizhen thank you very much for catching these typos!

Every little bit counts. Typos can be distracting to readers; removing them ticks up readability. 🙏

File: pkg/texttemplate/metas.go
Comment (jtigger) [inline pkg/texttemplate/metas.go:13]: Our linter only checks modified lines. Fixing this typo triggered the check for the presence of a comment on an exported identifier.

Here's a reasonable comment (or feel free to write your own).

```suggestion

// ShouldTrimSpaceLeft indicates whether leading spaces should be removed (because the left-trim token, `-`, was present)
```

**Review by jtigger — CHANGES_REQUESTED** [review]
Oops!

I missed the linter's complaint about wanting a comment on an exported method.

~I've added the suggestion, @hezhizhen. Please accept the suggestion or adjust to your preference. 👍~

edit: nvm, I see the PR is editable by maintainers; will fix that for ya. 👍

**Review by jtigger — APPROVED** [review]
Added comment required by linter; LGTM.

**Comment by hezhizhen** [comment]
> Oops!
> 
> I missed the linter's complaint about wanting a comment on an exported method.
> 
> ~I've added the suggestion, @hezhizhen. Please accept the suggestion or adjust to your preference. 👍~
> 
> edit: nvm, I see the PR is editable by maintainers; will fix that for ya. 👍

Thanks.

**Pull request merged**
